src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -0,0 +1,224 @@
+          guid
+          name
+          tags {
+            key
+            values
+          }
+        }
+      }
+    }
+  }
+}
+
+```
+**Step 2: Associate the repository entity to your application entity**
+
+First find the GUID for the application you want to associate your repository too.  
+
+Parameters:
+* `sourceEntityGuid` - the entity GUID of the application
+* `targetEntityGuid` - the entity GUID of your Repository
+* `type` - always `BUILT_FROM`
+
+```
+mutation {
+  entityRelationshipUserDefinedCreateOrReplace(sourceEntityGuid: "", targetEntityGuid: "", type: BUILT_FROM) {
+    errors {
+      message
+      type
+    }
+  }
+}
+```
+
+To see all entities related to your Repository you can do a query like this:
+```
+{
+  actor {
+    entity(guid: "[YOUR_REPOSITORY_GUID]]") {
+      relatedEntities(filter: {direction: BOTH, relationshipTypes: {include: BUILT_FROM}}) {
+        results {
+          target {
+            entity {
+              name
+              guid
+              type
+            }
+          }
+          type
+        }
+      }
+      name
+      type
+      tags {
+        values
+        key
+      }
+    }
+  }
+}
+
+```
+
+**Step 3: Cleanup (if needed)**
+
+Delete a repository with the following query:
+```
+mutation DeleteRepository {
+  entityDelete(guids: "[ENTITY_GUID_HERE]]") {
+    deletedEntities
+    failures {
+      message
+      guid
+    }
+  }
+}
+
+```
+
+  </Collapser>
+
+</CollapserGroup>
+
+## Install APM agents with CodeStream [#install-agents]
+
+Requirements for installing New Relic APM agents via CodeStream:
+* [New Relic account](https://newrelic.com/signup)
+* [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+* [CodeStream and New Relic connection](#connect)
+* A supported language application codebase
+    * .NET
+    * Java
+    * Node.JS
+
+With CodeStream connected to New Relic, when you're working on an application's codebase that's not being monitored by New Relic, CodeStream will offer to instrument that application for you.
+
+Like [New Relic's guided install](/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview/), CodeStream will walk you through and automate all of the steps to installing the APM agent to start sending data to New Relic.
+
+## Dynamic logging with Go and Pixie [#dynamic-logging]
+
+Requirements for dynamically logging your Go method names in your IDE:
+* [New Relic account](https://newrelic.com/signup)
+* [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+* [CodeStream and New Relic connection](#connect)
+* [A Kubernetes cluster monitored by Pixie](/docs/auto-telemetry-pixie/install-auto-telemetry-pixie/)
+* An application written in Go
+
+You can use CodeStream to enable dynamic logging for your Pixie-monitored Go applications.
+
+For relevant projects, you can use CodeStream to dynamically log your Go method names in your IDE.
```

Comment:
I feel like we need more detail here, although I know that it's a challenge since we're just finalizing the UI today. At a minimum we'll want to guide the user to right-click on a method name and select "Pixie Dynamic Logging" from the context menu. And then we'll probably want a screenshot of what this looks like. The general idea of how this will work is outlined here:

https://docs.google.com/document/d/1qWko2LqM5WfVgp3FEpiZpCsB7HPXby66-Qi3lEI6vAM/edit?usp=sharing

Reply:
I'm removing this content. When this feature is released, I'll add it back (and more!)